data/mods/Aftershock/recipes/frankenstein_recipes.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "type": "recipe",
+    "result": "inactive_frankenstein",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_CBMS",
+    "skill_used": "electronics",
+    "skills_required": [ "firstaid", 8 ],
+    "difficulty": 7,
+    "time": 50000,
+    "reversible": true,
+    "decomp_learn": 5,
+    "book_learn": [ [ "cyrus'_notes", 7 ] ],
+    "using": [ [ "soldering_standard", 20 ] ],
+    "tools": [ [ [ "afs_solderers_item", -1 ], [ "toolset", -1 ] ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
```

Comment:
Seperate defined qualities aren't needed here, since the required tools have that quality already.

Reply:
Do think I should add tailoring kit tool? To represent the surgery aspects?

Reply:
You could specify CUT_FINE level 2 or 3, since the scalpel has that.